src/BitcoinHelpers.js

```diff
@@ -482,7 +482,7 @@ const BitcoinHelpers = {
     estimateFee: async function(tbtcConstantsContract) {
       return new BN(
         await tbtcConstantsContract.methods.getMinimumRedemptionFee().call()
-      ).muln(4)
+      ).muln(18)
```

Comment:
:sweat_smile: so I know we're just hacking here, but this is as easy as eg

```
> curl https://api.blockcypher.com/v1/btc/main
{
  "name": "BTC.main",
  "height": 654832,
  "hash": "0000000000000000000c5007a9f2421e75397db74c176fd0ffcf8aa25a12a177",
  "time": "2020-10-31T03:01:57.684544702Z",
  "latest_url": "https://api.blockcypher.com/v1/btc/main/blocks/0000000000000000000c5007a9f2421e75397db74c176fd0ffcf8aa25a12a177",
  "previous_hash": "000000000000000000052e9104d14ef3eeb200fe7d62cac4efe38933ec6181af",
  "previous_url": "https://api.blockcypher.com/v1/btc/main/blocks/000000000000000000052e9104d14ef3eeb200fe7d62cac4efe38933ec6181af",
  "peer_count": 1044,
  "unconfirmed_count": 68673,
  "high_fee_per_kb": 388739,
  "medium_fee_per_kb": 185825,
  "low_fee_per_kb": 80523,
  "last_fork_height": 650491,
  "last_fork_hash": "00000000000000000007a7691b799d895602ce6b1ccd303c6446703a41e6d6de"
}
```

if we want to actually get an estimated feerate in the meantime

Reply:
Ignoring this and merging for now

Reply:
But then when we update to support multiple Electrum backends, we also need to support multiple centralized APIs for fee rates, or at least warn users that we'll be hitting a single centralized one as a backstop. Neither feels great. I would rather find some good Electrum servers to move between or something of the sort.

TBD in a later PR though.

Reply:
I'll warn you that even with many Electrum servers, we'll likely *still* have to have a list of centralized services we hit up, perhaps randomized, and then user feerate setting... because Bitcoin-land just doesn't have robust infrastructure for this part.

But yeah I hear you on the complexity / wanting to avoid SPoF